fast_llm/csrc/data.cpp

```diff
@@ -27,7 +27,7 @@
 
 /*
  Helper methods for fast index mapping builds.
- Changes for Fast-LLM: Use int16 for dataset index, add verbose argument to build_sample_idx.
+ Changes for Fast-LLM: Use int16 for dataset index, add verbose argument to build_sample_idx, add build_sample_idx_padded
```

Comment:
I was hoping to get rid of this file... But I guess it's more difficult for `build_sample_idx_padded`